MSAL/src/native_auth/public/MSALNativeAuthUserAccountResult.swift

```diff
@@ -75,6 +78,7 @@ import Foundation
     ///   - forceRefresh: Ignore any existing access token in the cache and force MSAL to get a new access token from the service.
     ///   - correlationId: Optional. UUID to correlate this request with the server for debugging.
     ///   - delegate: Delegate that receives callbacks for the Get Access Token flow.
+    @available(*, deprecated, message: "Use 'getAccessToken' method from 'MSALNativeAuthPublicClientApplication' instance")
```

Comment:
I don't understand this message, is the method not in `MSALNativeAuthUserAccountResult`? Rather than `MSALNativeAuthPublicClientApplication`?

Reply:
Sorry, I changed the code base on the agreement we had in that meeting, but forgot to correct the deprecation comment.